control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1525,6 +1532,8 @@ func (r *reconciler) ensureCloudResourcesDestroyed(ctx context.Context, hcp *hyp
 	}
 	if !removed {
 		remaining.Insert("persistent-volumes")
```

Comment:
shouldn't we move the PVs up, so they don't need to wait on the ingress controllers?